SH4RE/SH4RE/CreateListingView.swift

```diff
@@ -87,7 +87,13 @@ struct CreateListingView: View {
                     .padding()
                 
                 Button(action: {
-                    print("Post tapped!")
+                    var listing_fields = ["Title": title, "Description" : description, "Price" : cost, "Category" : drop_down_selection]
+                    let document_id = documentWrite(collectionPath: "Listings",data:listing_fields)
+                    //TODO : increment images when we add ability to upload multiple
+                    let image_path = "listingimages/" + document_id + "/1.jpg"
+                    storageManager.upload(image: image, path: image_path)
```

Comment:
where will this image be stored, is it in cloud storage and not the database? If so, is there a way to relate the image ID to the database entry?

Reply:
The image can't directly be stored in the database so we're instead storing it inside firebase cloud storage. The image path with upload to the database in the next line is the location of where the image is stored. So If we need the image we first get the path then request the image